test/azul_test_case.py

```diff
@@ -3,12 +3,21 @@
 from unittest.mock import (
     patch,
 )
+import warnings
 
 import boto3.session
 from botocore.credentials import Credentials
 import botocore.session
 
 
+class AzulTestCase(TestCase):
+
+    @classmethod
+    def setUpClass(cls) -> None:
+        super().setUpClass()
+        warnings.filterwarnings('ignore', 'unclosed.*<ssl.SSLSocket.*>', ResourceWarning)
```

Comment:
I think this line makes more sense in the second commit. First commit refactors, second commit adds changes.

Reply:
Done